docs/source/conf.py

```diff
@@ -81,7 +81,7 @@
     "paddlehub",
     "paddlehub.module.manager",
     "paddlehub.server.server",
-    "pickable_model",
+    "picklable_model",
```

Comment:
```suggestion
```
You don't have to mock this, since this section only applies to external library that might not be available for sphinx. Since `pickable_model` is an internal bentoml concept, no need to put under here